aws/windows-agent/README.md

```diff
@@ -1,20 +1,16 @@
-# ALPHA STATE - DO NOT USE THIS EXAMPLE
+# Windows Agent support (Alpha state example)
 
 This is an early example how to use and integrate windows agents. If you're not
 an active developer of this project we do not suggest to use it.
 
-## Terraform usage
+## Usage
+In order to get Windows instance span up, use latest `main.tf` and specify number of Windows Agents at `num` field.
```

Comment:
In order to spin up a DC/OS cluster with extra Windows agents, use the latest `main.tf` and specify number of Windows Agents at `num` field.